### PR TITLE
fix(css): step-error and step-killed padding

### PR DESCRIPTION
**Pull request by davidvader**
removing unused classes
```css
.step-error .message,
.step-skipped .message
```
and moving padding to step-killed
(in preparation for #197 )

### CODE REVIEW

**Review by jbrockopp — APPROVED** [review]
LGTM

File: src/elm/Pages/Build/View.elm [hunk 1]
```diff
@@ -592,7 +592,7 @@ stepFollowButton stepNumber following =
 stepError : Step -> Html msg
 stepError step =
     div [ class "step-error", Util.testAttribute "step-error" ]
-        [ span [ class "label" ] [ text "error:" ]
+        [ span [] [ text "error:" ]
         , span [ class "message" ]
```
Comment (wass3r) [inline src/elm/Pages/Build/View.elm:596]: can we get rid of this `span` then?

File: src/elm/Pages/Build/View.elm [hunk 2]
```diff
@@ -637,7 +637,7 @@ viewError build =
     case build.status of
         Vela.Error ->
             div [ class "error", Util.testAttribute "build-error" ]
-                [ span [ class "label" ] [ text "error:" ]
+                [ span [] [ text "error:" ]
                 , span [ class "message" ]
```
Comment (wass3r) [inline src/elm/Pages/Build/View.elm:641]: can we get rid of this `span` then?

**Review by wass3r — COMMENTED** [review]
i don't believe https://github.com/go-vela/ui/blob/d5a4e8a526314682637e99082c672e6abc543453/src/scss/_main.scss#L436 is used either, though i only did a cursory check.

**Pull request closed without merging**